README.md

```diff
@@ -115,6 +115,15 @@ A simple wrapper for the Jaeger exporter of [opentelemetry-java](https://github.
 ### More options
 For more options see [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation#configuration-parameters-subject-to-change)
 
+## Span tags the Splunk distribution automatically creates
+
+This Splunk distribution automatically adds context to `SERVER` spans for application servers with span tags. These are the span tags:
```

Comment:
maybe `adds context to ... spans for known application servers using span tags`?